spice/training/actions.py

```diff
@@ -284,6 +287,24 @@ def upload_models(self):
             status="UPLOADING",
         )
 
+        # Copies current training_round_model_repo config files (e.g. tokenizer, vocab)
+        # into model cache for upload to s3 and hf 
+        def copy_hf_to_cache(src_dir, dst_dir, ignore_files=["config.json", "pytorch_model.bin"]): # noqa
+            for file in os.listdir(src_dir):
+                if file in ignore_files:
+                    continue
+
+                src_path = f"{src_dir}/{file}"
+                dst_path = f"{dst_dir}/{file}"
+                shutil.copy2(src_path, dst_path)
```

Comment:
use path lib copy of shutil. https://stackoverflow.com/questions/33625931/copy-file-with-pathlib-in-python